websites/es/examples/headings.md

```diff
@@ -21,12 +21,12 @@ $web = new \spekulatius\phpscraper();
 /**
  * Navegue hasta la página de pruebas. Contiene:
  *
- * <title>¡Estamos probando aquí!</title>
```

Comment:
These are fixed in the test pages and we can't translate them, can you double check we are good with German too for this?

Reply:
Oh yes, I think we would need to revert them for `de` as well. I would do it first as we are having `de` version live as well

Reply:
It's alright - just fix it for DE and ES here. It isn't a big problem

Reply:
I've found some more places like this, have a look at the comments :)

Reply:
Onto that
I have understood it and will fix them all, together 👍🏼